spec/Powertrain/Battery.vspec

```diff
@@ -134,9 +134,34 @@ Charging.IsChargingCableConnected:
 Charging.ChargePlugType:
   datatype: string
   type: attribute
-  default: ccs
-  allowed: [ 'type 1', 'type 2', 'ccs', 'chademo' ]
-  description: Type of charge plug available on the vehicle (CSS includes Type2).
+  allowed: [
+    IEC_TYPE_1_AC,
+    IEC_TYPE_2_AC,
+    IEC_TYPE_3_AC,
+    IEC_TYPE_4_DC,
+    IEC_TYPE_1_CCS_DC,
+    IEC_TYPE_2_CCS_DC,
+    TESLA_ROADSTER,
+    TESLA_HPWC,
+    TESLA_SUPERCHARGER,
+    GBT_AC,
+    GBT_DC,
+    OTHER
+    ]
+  description: Type of charge plug (charging connector) available on the vehicle.
+               IEC types refer to IEC 62196,  GBT refers to  GB/T 20234.
+  comment:     IEC_TYPE_1_AC refers to Type 1 as defined in IEC 62196-2. Also known as Yazaki or J1772 connector.
+               IEC_TYPE_2_AC refers to Type 2 as defined in IEC 62196-2. Also known as Mennekes connector.
```

Comment:
Should this multiline not use the Literal format, or is also this way OK according to spec?
Ref literals:  https://yaml.org/spec/1.2-old/spec.html#id2795688

Reply:
As I understand this format is "Yaml Plain style" see example 7.12 in  https://yaml.org/spec/1.2.2/#733-plain-style.
This is also described as "plain" in https://yaml-multiline.info/

So my interpretation is that this is valid Yaml.

With this format the output does not contain the newline used e.g.between line 151 and 152 above, i.e. it get merged to a single line. If using literal format (`|`) line breaks will be kept. We seem to support that correctly for vspec2yaml but not for vspec2csv as we then generate the linebreak rather than some form of escape (`\n`). But that is rather a bug in csv. In general I think it would be good if someone had time to check how the various tools handle various Yaml syntaxes, but that is not to be handled by this PR

Reply:
Looks like it's valid YAML format then. OK, question answered.